default.nix

```diff
@@ -32,9 +32,15 @@ let
     llvmPackages_12.bintools
     rustc-nightly
     cargo-nightly
-    xargo
     wasmtime
     rust-bindgen
+    python3
+    tree
```

Comment:
Leftover from earlier revision?

Reply:
I was using this for debugging. Removed now.